rust/abacus-base/src/contract_sync/metrics.rs

```diff
@@ -5,15 +5,31 @@ use std::sync::Arc;
 /// Struct encapsulating prometheus metrics used by the ContractSync.
 #[derive(Debug, Clone)]
 pub struct ContractSyncMetrics {
-    /// Most recently indexed block height (label values differentiate checkpoints
-    /// vs. messages)
+    /// Most recently indexed block height (label values differentiate
+    /// checkpoints vs. messages)
```

Comment:
Also we are not actually indexing checkpoints are we?

Reply:
It seems we are not. Updated.